### PR TITLE
fix: build issue

### PR DESCRIPTION
**Pull request by NikitK-deriv**

### CODE REVIEW

File: src/growthbook.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
-import { GrowthBook } from '@growthbook/growthbook-react'
+import { GrowthBook } from '@growthbook/growthbook'
 import * as RudderAnalytics from 'rudder-sdk-js'
 
 export type GrowthBookType = GrowthBook
```
Comment (markwylde-deriv) [inline src/growthbook.ts:4]:
```suggestion
export type GrowthBookType = GrowthBook
```

😕 Can we not do this, and just use `GrowthBook` directly?

File: src/growthbook.ts [hunk 2]
```diff
@@ -11,7 +16,7 @@ export type AttributesTypes = {
     device_type: string
 }
 export class Growthbook {
-    GrowthBook
+    GrowthBook: GrowthBookType
```
Comment (markwylde-deriv) [inline src/growthbook.ts:19]:
```suggestion
    GrowthBook
```

File: __tests__/rudderstack.spec.ts
Comment (markwylde-deriv) [inline __tests__/rudderstack.spec.ts:1]: 😢

File: src/growthbook.ts (hunk 1 above)
Comment (NikitK-deriv) [reply src/growthbook.ts:4]: Yes, I thought we can use export this type for instances but I hope it'll rare case

File: __tests__/rudderstack.spec.ts
Comment (NikitK-deriv) [reply __tests__/rudderstack.spec.ts:1]: returned test back

File: src/growthbook.ts [hunk 3]
```diff
@@ -11,7 +15,7 @@ export type AttributesTypes = {
     device_type: string
 }
 export class Growthbook {
-    GrowthBook
+    GrowthBook: GrowthBook
```
Comment (markwylde-deriv) [inline src/growthbook.ts:18]:
```suggestion
    GrowthBook
```

I think we don't need to change this line.

**Review by markwylde-deriv — APPROVED** [review]

File: src/growthbook.ts (hunk 3 above)
Comment (NikitK-deriv) [reply src/growthbook.ts]: remove a type
and yes, this type wasn't helpful, thanks you for repeating the comment, I didn't get you first time

**Review by markwylde-deriv — APPROVED** [review]

**Pull request merged**